generic/tests/cfg/pktgen_perf.cfg

```diff
@@ -22,6 +22,10 @@
         - @default:
             only bridge
             test_vm = yes
+        - l1_vp_vdpa:
```

Comment:
The variant name is not quite suitable, if a VM executes this scenario, then it's l2. so how about vm(guest)_vp_vdpa

Reply:
For this case, it only supports on L1, if test with L2, it need to add other steps.

Reply:
The case focuses on guest vp_vdpa, I don't think we need to clarify it's L1 or L2.

Reply:
I updated.